helmfile.d/snippets/env.gotmpl

```diff
@@ -20,6 +20,8 @@ environments:
   default:
     missingFileHandler: Debug
     values:
+      - teamConfig:
```

Comment:
Why create a new row? Please merge with the other

Reply:
Because if it is not set, it defaults back into this. If it is set, it will be merged on top of this. Makes the smallest, cleanest change.

Reply:
you did not get what I meant...let me fix it in the code